playbooks/generic.yml

```diff
@@ -3,7 +3,7 @@
   hosts: "{{ hosts_to_deploy | default('local') }}"
   become: true
   roles:
-    - pkg_mgmt/pip
```

Comment:
please uncomment and test again, paste output from azure cli to see that it's working

```
az version
```